### PR TITLE
Init

### PR DESCRIPTION
**Pull request by AmsTaFFix**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+git:
+  depth: 1
+
+before_script:
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.40
+
+script:
+  - golangci-lint run --modules-download-mode=vendor --deadline=5m
+  - go test -v -race ./...
```
Comment (liderman) [inline .travis.yml:20]: Please add a blank line at the end

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
-# uuid-msgpack
-Library for integrating github.com/google/uuid with gopkg.in/vmihailenco/msgpack.v2.
+# UUID msgpack
+Library to integrate github.com/google/uuid with gopkg.in/vmihailenco/msgpack
+
+
+Installation
+------------
+	go get github.com/citilinkru/camunda-client-go/v2
```
Comment (liderman) [inline README.md:7]: What "camunda-client-go"?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,75 @@
...
+
+Example
+-------
+```go
+package main
+
+import (
+	_ "code.citik.ru/gobase/uuid-msgpack" // Обязательно нужно сделать импорт!
```
Comment (liderman) [inline README.md:15]: Please fix for public import github.com...
and translate the Russian comment into English

File: .travis.yml (hunk 1 above)
Comment (liderman) [reply .travis.yml:20]: Need to make migration to Github Actions: https://docs.github.com/en/actions
TravisCI is closed for open source projects.

**Review by liderman — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AmsTaFFix) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (AmsTaFFix) [reply README.md]: fixed

File: .travis.yml (hunk 1 above)
Comment (AmsTaFFix) [reply .travis.yml]: done

**Pull request merged**